package.json

```diff
@@ -13,6 +13,7 @@
     "@typescript-eslint/eslint-plugin": "^2.8.0",
     "@typescript-eslint/parser": "^2.8.0",
     "@vtex/react-csv-parse": "^3.0.2",
+    "antd": "4.0.3",
```

Comment:
I thought this lib would cause conflict with our current design

Reply:
Ahhh ok, do you want me to remove it? I think it got added when I merged with dev

Reply:
Yeah, no worries, I'll remove it